tasks/dist/css

```diff
@@ -0,0 +1 @@
+node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
```

Comment:
In https://docs.npmjs.com/cli/run-script, it sounded like if the `devDependency` is there, `npm run` will do that for us.

> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix. For example, if there is a devDependency on tap in your package, you should write:

Reading that, maybe I just need to add `node-sass` to `devDependency`? Again, I've no idea so I'm likely wrong :grin:.